src/github/nag.rs

```diff
@@ -185,7 +130,147 @@ fn update_proposal_review_status(proposal_id: i32) -> DashResult<()> {
     Ok(())
 }
 
+fn update_poll_response_status(poll_id: i32) -> DashResult<()> {
```

Comment:
This function seems pretty well scoped to the `Poll` type -- what do you think about making it a method and leaving the exact query used to retrieve the `Poll` flexible?